test/service/test_request_validation.py

```diff
@@ -180,13 +185,13 @@ def test_bad_sort_facet_but_valid_filter_facet_of_file(self):
         url = self.base_url + '/index/files'
         params = {
             'size': 15,
-            'sort': 'bad-facet',
-            'order': 'asc',
+            'sort': 'bad-facet', ''
+                                 'order': 'asc',
```

Comment:
Typo?

Reply:
yeah not sure how its formated so nicely. waiting on tests my bad.